c7n/resources/route53.py

```diff
@@ -208,6 +208,65 @@ def process_resource_set(self, client, domains, keys):
                 TagsToDelete=keys)
 
 
+@ResourceRecordSet.action_registry.register('remove-recordset')
```

Comment:
re simplifying the name we don't need to duplicate the type in the action name, its already scoped. `delete` seems to be the closest.

Reply:
changed the action name from '**remove-recordset**' to '**delete**'